mapie/conformity_scores/residual_conformity_scores.py

```diff
@@ -132,3 +141,275 @@ def get_estimation_distribution(
         """
         self._check_predicted_data(y_pred)
         return np.multiply(y_pred, np.add(1, conformity_scores))
+
+
+class FittedResidualNormalisingScore(ConformityScore):
+    # or ConformalResiudalFittingScore
+    # or NormalizedScore
```

Comment:
SUGGESTION: I can suggest using `ResidualConformityScore`.
```suggestion
class ResidualConformityScore(ConformityScore):
```

Reply:
Done